src/main/kotlin/io/github/mojira/arisa/infrastructure/Jira.kt

```diff
@@ -61,6 +72,22 @@ fun updateCommentBody(jiraClient: JiraClient, comment: Comment, newValue: String
     }
 }
 
+fun restrictComment(jiraClient: JiraClient, comment: Comment, group: String, body: String? = comment.body) = runBlocking {
+    Either.catch {
```

Comment:
You can do theComment.update() and change the visibility

Reply:
This method does not exist in the version being used.

Reply:
We should be able to update it by just changing the version in build.gradle

Reply:
The latest version available at maven seems to be https://mvnrepository.com/artifact/net.rcarz/jira-client/0.6.3-IHTSDO.
This version does include `Comment.update()` but still no visibility.

Reply:
ugh, I checked the repo, apparently they never published a newer version. The other libs I could find was the official one that relies on everything being a rest client call, so I would just leave it like this until we can find another replacement library

Reply:
Or build the lib from sourve

Reply:
> Or build the lib from sourve
Not sure

Reply:
> Or build the lib from sourve

Not sure how this would work with gradle.

Reply:
Done using jitpack